<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RGBLightClientClass</class>
 <widget class="QMainWindow" name="RGBLightClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RGBLightClient</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RGBLightClient.qrc">
    <normaloff>:/RGBLightClient/SystrayIcon</normaloff>:/RGBLightClient/SystrayIcon</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>灯效</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,0">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>9</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>设备列表</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>刷新</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_detailPage"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>设置</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RGBLightClient.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>RGBLightClientClass</receiver>
   <slot>onRefreshButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>538</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>RGBLightClientClass</receiver>
   <slot>onListWidgetSelectionChanged(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onRefreshButtonClicked()</slot>
  <slot>onListWidgetSelectionChanged(QListWidgetItem*)</slot>
 </slots>
</ui>
